db/migrations/public/20231127222209_create-licences-view.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const viewName = 'licences'
+
+exports.up = function (knex) {
+  return knex
+    .schema
+    .createView(viewName, (view) => {
+      // NOTE: We have commented out unused columns from the source table
+      view.as(knex('licences').withSchema('water').select([
+        'licence_id AS id',
+        'region_id',
+        'licence_ref',
+        'is_water_undertaker',
+        'regions',
+        'start_date',
+        'expired_date',
+        'lapsed_date',
+        'revoked_date',
+        'suspend_from_billing',
+        // 'is_test',
+        'include_in_supplementary_billing',
+        'include_in_sroc_supplementary_billing',

```

Comment:
Not sure this suggestion is necessarily better. But can't afford to miss the opportunity to make these more usable in our code! How about

```suggestion
        'include_in_supplementary_billing AS include_in_presroc_billing',
        'include_in_sroc_supplementary_billing AS include_in_sroc_billing',

```